css/components/snackbar.css

```diff
@@ -0,0 +1,16 @@
+.hide{
+  display: none;
+}
+
+.snackbar{
+  width: 20rem;
+  background-color: var(--sec-col);
+  color: var(--prim-col);
+  padding: 0.5rem 1rem;
+  border-radius: 0.5rem;
```

Comment:
It is advisable to use px when rem value is less than 1.

Reply:
Yes use px for smaller than 1rem